infrateam_first_responder.md

```diff
@@ -68,14 +68,15 @@ Note that you will need to be sure you can ssh into each of the VMs from whereve
 
 ##### Code that isn't a Ruby Application
 
-We have codebases to maintain that aren't Ruby applications or gems. We have not yet settled on a long-term method for dealing with these:
+We have codebases to maintain that aren't Ruby applications or gems. We have not yet settled on a long-term method for dealing with all of these:
 
-- javascript apps with npm updates:
-  - https://github.com/sul-dlss/access-update-scripts has code to create PRs for npm package updates, but it currently only works for the sul-dlss github organization. All the sinopia code is in the LD4P github organization. There is a [pending pull request](https://github.com/sul-dlss/access-update-scripts/pull/82) that will address this.
-- java code (note that we are down to two Java repositories in our platform: [suri2](https://github.com/sul-dlss/suri2) (which is slated to be replaced with the Rails-based [suri_rails](https://github.com/sul-dlss/suri_rails) in 2020), and [etd-reporter](https://github.com/sul-dlss/etd-reporter).)
+- JavaScript code, which is now updated by the same [access-update-scripts](https://github.com/sul-dlss/access-update-scripts) project as our Ruby scripts.
+- Java code (note that we are down to two Java repositories in our platform: [dlss-wowza](https://github.com/sul-dlss/dlss-wowza/) (our user authentication plugin for Wowza, the media streaming software we use), and [etd-reporter](https://github.com/sul-dlss/etd-reporter).)
 - Go projects (such as various RIALTO components)
```

Comment:
This line can go as well, as we have no Go projects?

Reply:
ah, didn't realize that, fixed, thanks!